pkg/sidecar/configs.go

```diff
@@ -152,8 +152,7 @@ func (cfg *Config) MysqlDSN() string {
 
 // IsFirstPodInSet returns true if this pod has an ordinal of 0, meaning it is the first one in the set
 func (cfg *Config) IsFirstPodInSet() bool {
-	ordinal := getOrdinalFromHostname(cfg.Hostname)
-	return ordinal == 0
+	return strings.HasSuffix(cfg.Hostname, "-0")
```

Comment:
I think you should use the `getOrdinalFromHostname()` function, much more now when you made it better. 😄

Reply:
I just think it would improve the performance a bit if it is judged directly, it is not too important. Using `getOrdinalFromHostname` would also work

Reply:
Using `HasSuffix` will improve performance, using `getOrdinalFromHostname` will have better readability, which one do you think is better?

Reply:
I opt for readability, the performance gain is insignificant in this case